css/full-width-pics.css

```diff
@@ -64,13 +64,177 @@ body {
   justify-content: flex-start;
 }
 
-.name-box{
+/*.name-box{
   display: flex;
   flex-direction: column;
-}
+}*/
 
 .profileimg{
-  width:auto;
-  height:120px;
+  width:262px;
+  height:262px;
   margin-right:20px;
 }
+
+.article{
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.article .title {
+  flex: 0 1 100%;
+  font-weight: bold;
+  font-size: 18px;
+  color: black;
+}
+
+.article .title #popup{
+  flex: 0 1 100%;
+  font-weight: bold;
+  font-size: 18px;
+  color:white;
+  
+}
+
+.article .title:hover #popup{
+  flex: 0 1 100%;
+  font-weight: bold;
+  font-size: 18px;
+  color: black;
+}
+
+.article .title:focus #popup{
+  flex: 0 1 100%;
+  font-weight: bold;
+  font-size: 18px;
+  color: black;
+}
+
+.page-content{
+  min-height: calc(100vh - 70px);
+}
+
+.footer{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: #343a40;
+  height: 56px;
+}
+
+.footer-container{
+  padding-left: 20px;
+}
+
+.footer-text{
+  color: white;
+  font-size: 15px;
+  font-family: sans-serif;
+}
+
+/********* People Page CSS ***********/
+
+.faculty-container{
```

Comment:
CSS allows for bad patterns to develop (just like any programming language does), and these are called code smells. The repetition of code within the `.faculty-container`, `.grad-container`, `.undergrad-container`, and `.alumni-container` classes is indicative of the [**Duplicate Code**](https://sourcemaking.com/refactoring/smells/duplicate-code) smell. 

I would recommend that you instead, define the base for all of these classes via the following combined CSS rule:
```css
.faculty-container, .grad-container, .undergrad-container, .alumni-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: flex-start;
}
```
This combined rule has the same effect as the separate duplicate rules, but now there is only one set of code that defines the baseline options for these containers (and therefore any updates for the baseline only need to occur in this one location). This does not effect the specialization rules that are included below these container rules.